components/brave_wallet_ui/components/desktop/inspect-nfts/components/info-tooltip/info-tooltip.style.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+
+export const StyledWrapper = styled.div`
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  z-index: 3;
+  background-color: #1E2029;
```

Comment:
Do we have a `theme` variable for this? if not should we add a `(prefers-color-scheme: dark)` here?